### PR TITLE
[ASAP-604] - format discussion replies

### PR DESCRIPTION
**Pull request by AkosuaA**

### CODE REVIEW

File: packages/react-components/src/molecules/__tests__/Discussion.test.tsx [hunk 1]
```diff
@@ -60,3 +62,77 @@ it('removes reply modal when user clicks cancel button', () => {
...
+      const discussion = createDiscussionResponse(message, replies);
+      getDiscussion.mockReturnValue(discussion);
+
+      const { getByLabelText, getByText, queryByText, debug } = render(
+        <Discussion {...propsWithReplies} />,
+      );
+
+      expect(queryByText(/test reply 1/i)).not.toBeInTheDocument();
+      userEvent.click(getByLabelText(/\+1/));
+      debug(undefined, 5000000);
```
Comment (gabiayako) [inline packages/react-components/src/molecules/__tests__/Discussion.test.tsx:134]:
```suggestion
```

File: packages/react-components/src/molecules/__tests__/Discussion.test.tsx [hunk 2]
```diff
@@ -60,3 +62,77 @@ it('removes reply modal when user clicks cancel button', () => {
 
   expect(queryByText(/Reply to quick check/i)).not.toBeInTheDocument();
 });
+
+describe('when there are replies', () => {
+  describe('when collapsed', () => {
```
Comment (gabiayako) [inline packages/react-components/src/molecules/__tests__/Discussion.test.tsx:67]: We don't need this to be wrapped in this `when collapsed` describe, right?

**Review by gabiayako — APPROVED** [review]
nice! 👏👏👏

File: packages/react-components/src/molecules/__tests__/Discussion.test.tsx (hunk 2 above)
Comment (AkosuaA) [reply packages/react-components/src/molecules/__tests__/Discussion.test.tsx:67]: Yea, agreed. Removing it

**Pull request merged**